docker/openbao-entrypoint.sh

```
@@ -0,0 +1,426 @@
+    code_signing_flag=false \
+    email_protection_flag=false \
+    key_type=rsa \
+    key_bits=2048 \
+    key_usage=\"DigitalSignature,KeyEncipherment,KeyAgreement\" \
+    ext_key_usage=\"ServerAuth,ClientAuth,OCSPSigning\" \
+    use_csr_common_name=true \
+    use_csr_sans=true \
+    copy_extensions=true \
+    allowed_extensions=\"*\" \
+    basic_constraints_valid_for_non_ca=true \
+    max_ttl="$OPENBAO_MAX_LEASE_TTL" \
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)